cypress/integration/recordings.spec.js

```diff
@@ -3,7 +3,7 @@
  */
 context('Recordings', function () {
   describe('On the search page', () => {
-    it('should display for words  on the search page', () => {
```

Comment:
nitpick, "on the search page" was already inside `describe`

Reply:
Haha, good point!